### DISCUSSION TITLE
Wiki in multiple languages

### DISCUSSION THREAD

**Discussion by eco-star** [linuxmint/discussions#204] [Ideas]
Hi guys,

I looked at the Wiki and thought about an improvement in documentation.
It would be awesome to provide the Wiki in other languages aswell and let the community improve the instructions in their own native language. In addition to that, the Wiki-page could be presented as a menu-button in linuxmint.com aswell.

As I can not code and help with code-contribution, I could help in instruction-translation and creation in german and also a bit in brazilian portuguese.

Thanks for your attention and your time in maintaining and improving Mint! :)

**Comment by ghost** [comment]
I am the one that made [this](https://github.com/orgs/linuxmint/discussions/210) discussion, regarding a general improvement in the user documentation. Since this one is mainly for translations, I thought I would add that, currently, the [Translation Guide](https://linuxmint-translation-guide.readthedocs.io/en/latest/) does not reference documentation, and it also does not seem to be on [Launchpad](https://translations.launchpad.net/linuxmint) (although I did not have the time to look for it elsewhere).

**Reply by eco-star** [reply]
Hey, thanks for your suitable additions :)
